broker/bpmn.go

```diff
@@ -52,6 +54,15 @@ func runBrokerBpmn(policy *models.Policy, flowKey string) *bpmn.State {
 
 	// TODO: fix me - maybe get to/from/cc from setting.json?
 	switch flowKey {
+	case leadFlowKey:
+		flow = setting.LeadFlow
+		toAddress = mail.GetContractorEmail(policy)
+		switch channel {
+		case models.AgentChannel:
+			ccAddress = mail.GetAgentEmail(policy)
+		default:
+			ccAddress = mail.Address{}
+		}
 	case proposalFlowKey:
 		flow = setting.ProposalFlow
 		toAddress = mail.GetContractorEmail(policy)
```

Comment:
If in the new proposal flow we are not sending emails (for now), maybe we should set all address to empty ones.